test/assets/video.html

```diff
@@ -0,0 +1,16 @@
+<html> 
+<body> 
+
+<video width="400" controls>
+  <source src="movie.mp4" type="video/mp4">
+  <source src="movie.ogv" type="video/ogg">
```

Comment:
Last time I tried this, our WebKit on mac did not play the ogg. It does a range request that's not supported by our test server.

Reply:
That's unfortunate, marking it as failing in WebKit for now anyway.